cmd/checks_windows.go

```diff
@@ -83,13 +83,13 @@ func processCheck(check *check, checkType, arg1, arg2, arg3 string) bool {
 		if check.Message == "" {
 			check.Message = "Security policy option " + arg1 + " is \"" + arg2 + "\""
```

Comment:
We may want to add some additional logic here to change the message depending on whether or not they're using `arg3`.

Reply:
I thought about it--

but then I realized that would be like, another 30 lines of code

Reply:
I'm in the code though, so I'll do it anyway